hieradata/role/webserver.yaml

```diff
@@ -2,6 +2,8 @@
 psick::profiles::linux_classes:
   webserver: 'psick::apache::tp'
 
```

Comment:
Need to add:
psick::base::windows_classes:
  packages: psick::packages
So it can use chocolatery as a package manager

Reply:
right.
this could be added direclty in common.yaml https://github.com/graduway/psick/blob/production/hieradata/common.yaml#L57 if we want to include the  class psick::packages on all the windows nodes

Reply:
I'm doing that, and refactoring the roles.